### ISSUE TITLE
fasterq-dump --concatenate-reads does not run

### ISSUE THREAD

**Issue by zyqfrog10** [ncbi/sra-tools#279]
Hello,

fasterq-dump --concatenate-reads does not run entirely. help information will pop out instead.

Thanks,
YZ
```
> fasterq-dump SRR341578 --concatenate-reads 

Usage:
  fasterq-dump <path> [options]

Options:
  -o|--outfile                     output-file 
  -O|--outdir                      output-dir 
  -b|--bufsize                     size of file-buffer dflt=1MB 
  -c|--curcache                    size of cursor-cache dflt=10MB 
  -m|--mem                         memory limit for sorting dflt=100MB 
  -t|--temp                        where to put temp. files dflt=curr dir 
  -e|--threads                     how many thread dflt=6 
  -p|--progress                    show progress 
  -x|--details                     print details 
  -s|--split-spot                  split spots into reads 
  -S|--split-files                 write reads into different files 
  -3|--split-3                     writes single reads in special file 
  --concatenate-reads              writes whole spots into one file 
  -Z|--stdout                      print output to stdout 
  -f|--force                       force to overwrite existing file(s) 
  -N|--rowid-as-name               use row-id as name 
  --skip-technical                 skip technical reads 
  --include-technical              include technical reads 
  -P|--print-read-nr               print read-numbers 
  -M|--min-read-len                filter by sequence-len 
  --table                          which seq-table to use in case of pacbio 
  --strict                         terminate on invalid read 
  -B|--bases                       filter by bases 
  -A|--append                      append to output-file 
  --ngc <PATH>                     PATH to ngc file 

  -h|--help                        Output brief explanation for the program. 
  -V|--version                     Display the version of the program then 
                                   quit. 
  -L|--log-level <level>           Logging level as number or enum string. One 
                                   of (fatal|sys|int|err|warn|info|debug) or 
                                   (0-6) Current/default is warn 
  -v|--verbose                     Increase the verbosity of the program 
                                   status messages. Use multiple times for more 
                                   verbosity. Negates quiet. 
  -q|--quiet                       Turn off all status messages for the 
                                   program. Negated by verbose. 
  --option-file <file>             Read more options and parameters from the 
                                   file. 

/home/zhany/tools/sratoolkit.2.10.2-centos_linux64/bin/fasterq-dump : 2.10.2
```

**Comment by skripche** [comment]
Thank you for reporting the problem and the same issue is in 2.10.3.

**Comment by glev-doug** [comment]
This issue appears to be persistent in 2.10.5 as well FYI.

**Comment by jpuritz** [comment]
Also in 2.10.6

**Comment by kwrodarmer** [comment]
Thank you for this. We'll take it back into the shop.

**Comment by gfairchild** [comment]
I'm experiencing this with 2.10.8 as well.

**Comment by monfortm** [comment]
As already reported, I also get the issue with latest version 2.10.8. The command runs well without the `--concatenate-reads` parameter. However when I add it to the command line I get the usage/help back, exit status 0

**Comment by durbrow** [comment]
Thank you all for the prodding. This will be [fixed](https://github.com/ncbi/sra-tools/commit/e2b789be04d84945f2c636197339452df11a839c) really soon.

**Comment by klymenko** [comment]
@zyqfrog10, the bug was fixed in [our latest 2.10.9 release](https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software).
Please confirm.
